tuxemon/core/states/combat/combat_animations.py

```diff
@@ -468,9 +466,9 @@ def animate_parties_in(self):
         back_island = self.load_sprite('gfx/ui/combat/' + self.graphics['island_back'],
                                        bottom=opp_home.bottom + y_mod, right=0)
 
-        monster1 = self.load_sprite(right_monster.front_battle_sprite,
-                                    bottom=back_island.rect.bottom - scale(12),
-                                    centerx=back_island.rect.centerx)
+        monster1 = right_monster.get_sprite("front",
+                                            bottom = back_island.rect.bottom - scale(12),
```

Comment:
don't add spaces between then the `=` and words when the `=` is in a function/method call

Reply:
`self.sprites.add(monster1)`